server/game/cards/12-KotI/GreatWyk.js

```diff
@@ -0,0 +1,47 @@
+const DrawCard = require('../../drawcard.js');
+
+class GreatWyk extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction ({
+            when: {
+                onCardEntersPlay: event=>(
+                    event.card.getType() === 'character' &&
+                    event.card.controller === this.controller &&
+                    event.originalLocation === 'dead pile'
+                )}
```

Comment:
Spacing here is not quite in convention with the rest of the codebase. The closing `}` should come at the same indentation as the `when`. So more like:
```javascript
            when: {
                onCardEntersPlay: event => (
                    event.card.getType() === 'character' &&
                    event.card.controller === this.controller &&
                    event.originalLocation === 'dead pile'
                )
            },
```